templates/head-meta.html

```diff
@@ -10,7 +10,7 @@
 
 <link rel="publisher" href="https://plus.google.com/107187849809354688692">
 <link rel="shortcut icon" href="/images/logos/p-logo-32.png">
-<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
+<link rel="preload" href="//fonts.googleapis.com/css?family=Source+Code+Pro:400,700|Roboto+Slab:400,700">
```

Comment:
I saw an error message in the console about those font resources being loaded via `preload` but not actually used. My understanding is similar to what @keanulee said, in that you need to explicitly use `<link rel="stylesheet">` to put the fonts to use. `preload`'s benefit is to kick off the request to start downloading the resource earlier when it might not normally be loaded right away: https://developers.google.com/web/updates/2016/03/link-rel-preload